test_variable_combinations.py

```diff
@@ -64,6 +56,7 @@ def test_missing_passing(left_out):
         assert isclose(removed[variable], getattr(lens, variable))
 
 
+@pytest.mark.skip("At the moment, there aren't any failing combinations.")
```

Comment:
The test is skipped automatically when the `combinations_failing` is empty so you don't need this decorator.